server/routes/api/auth.ts

```diff
@@ -168,8 +168,13 @@ router.post("auth.delete", auth(), transaction(), async (ctx: APIContext) => {
     }
   );
 
+  const authentications = await user.$get("authentications");
```

Comment:
Yea, you're right – it would be better to find any OIDC authentication specifically rather than just use the first.

Reply:
Can there be several OIDC authentication for one user? or it's not a use case that can happen?

Reply:
Unlikely but technically possible if we can account for it